perun-core/src/test/java/cz/metacentrum/perun/core/bl/VosManagerBlImplUnitTest.java

```diff
@@ -0,0 +1,90 @@
+package cz.metacentrum.perun.core.bl;
+
+import cz.metacentrum.perun.core.api.*;
```

Comment:
We don't want to use wildcard imports. Please, try to adjust your IDE's settings.